ruby_exercises/mythical-creatures/spec/medusa_spec.rb

```diff
@@ -26,16 +27,40 @@
     medusa = Medusa.new('Cassiopeia')
     victim = Person.new('Perseus')
 
+    
     expect(victim.stoned?).to be false
     medusa.stare(victim)
     expect(victim.stoned?).to be true
   end
 
   it 'can only have three victims' do
-    # your code here
+    medusa = Medusa.new('Cassiopeia')
+    victim_1 = Person.new('Perseus')
+    victim_2 = Person.new('Persephone')
+    victim_3 = Person.new('Penelope')
+
+    medusa.stare(victim_1)
+    medusa.stare(victim_2)
+    medusa.stare(victim_3)
+
+    expect(medusa.three_victims?).to eq true 
```

Comment:
I might write the tests in a different way.
In this test, I'd check the count of statues for Medusa.
In the following test, I'd check the count, check that the first victim returns `false` for `stoned?` and that the other 3 victims return `true` for `stoned?`.